cluster-sync/ephemeral_provider.sh

```diff
@@ -49,10 +49,14 @@ function configure_storage() {
 }
 
 function configure_hpp() {
-  for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
-      ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo mkdir -p /var/hpvolumes"
-      ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo chcon -t container_file_t -R /var/hpvolumes"
-  done
+  if [[ $KUBEVIRT_PROVIDER =~ kind.* ]]; then
+      ./cluster-up/ssh.sh kind-1.28-control-plane mkdir -p /var/hpvolumes
+  else
+      for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
+         ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo mkdir -p /var/hpvolumes"
+         ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo chcon -t container_file_t -R /var/hpvolumes"
```

Comment:
Do you not need to apply the second command?

Reply:
Yes, it works fine in the Arm64 CI line.

Reply:
I get following error when trying to run this command:
```
root@kind-1:/# chcon -t container_file_t -R /var/hpvolumes
chcon: can't apply partial context to unlabeled file 'csi'
chcon: can't apply partial context to unlabeled file '/var/hpvolumes'
```
Maybe because my host system is ubuntu in which selinux does not apply?

Reply:
Yes something smells off with the selinux setup but I don't think it's worth blocking over this so it's okay

Reply:
You can actually remove the chcon it was needed for the legacy version of hpp, but since we only run the csi version this is not a concern anymore.

Reply:
Note applying it in the x86 case doesn't hurt anything. It is not needed though